src/common/sidebar/containers/SidebarTemplate.tsx

```diff
@@ -56,9 +55,6 @@ export function SidebarTemplate({ children, term, title }: PropsWithChildren<Sid
         <Flex minW="80%" overflow="auto" justifyContent="center" boxShadow="lg" zIndex={56}>
           {children}
         </Flex>
-        <Box pos="absolute" bottom="0" right="0" zIndex={999} p={25}>
```

Comment:
`Box` is still in the imports

Reply:
Ahaha, I swear I checked all of them this time. We should really catch this in the linter. Should I make an issue?

Reply:
Yeah, let's make an issue and I'll address it ASAP. This is definitely the most common warning and we've been meaning to fix it for a while but always seem to forget 😅